packages/expo-background-task/src/BackgroundTask.ts

```diff
@@ -0,0 +1,104 @@
+import { UnavailabilityError } from 'expo-modules-core';
+import * as TaskManager from 'expo-task-manager';
+
+import { BackgroundTaskOptions, BackgroundTaskStatus } from './BackgroundTask.types';
+import ExpoBackgroundTaskModule from './ExpoBackgroundTaskModule';
+
+// @needsAudit
+/**
+ * Returns the status for the Background Task API. On web, it always returns `BackgroundTaskStatus.Restricted`,
+ * while on native platforms it returns `BackgroundTaskStatus.Available`. There is
+ *
+ * @returns A BackgroundTaskStatus enum value or null if not available.
```

Comment:
```suggestion
 * @returns A BackgroundTaskStatus enum value or `null` if not available.
```

We should also format `null`.